File: Shedule-SPA/src/app/_models/problem.ts
import { ProblemPhoto } from './problemPhoto';

export interface Problem {
  id: number;
  type: string;
  degree: number;
  description: string;
  created: Date;
  timeHappened: Date;
  city: string;
  country: string;
  address: string;
  longitude: number;
  latitude: number;
  photoUrl: string;
  userId: number;
  photos?: ProblemPhoto[];
  // only for client
  // hovered?: boolean;
}

File: Shedule.API/Models/Problem.cs
using System;
using System.Collections.Generic;

namespace Shedule.API.Models
{
    public class Problem
    {
        public int Id { get; set; }
        public string Type { get; set; }
        public int Degree { get; set; }
        public string Description { get; set; }
        public DateTime Created { get; set; }
        public DateTime TimeHappened {get; set;}
        public string City { get; set; }
        public string Country { get; set; }
        public string Address { get; set; }
        public double Longitude { get; set; }
        public double Latitude { get; set; }
        public User User { get; set; }
        public int UserId { get; set; }
        public ICollection<ProblemPhoto> Photos { get; set; }
    }
}